charts/kuberpult/run-kind.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+#set -x
+
+# This script assumes that the docker images have already been built.
+# To run/debug/develop this locally, you probably want to run like this:
+# make clean; LOCAL_EXECUTION=true ./run-kind.sh
+
+cd "$(dirname $0)"
+
+#(make -C ../../services/cd-service/)
+
+# prefix every call to "echo" with the name of the script:
+function print() {
+  /bin/echo "$0:" "$@"
+}
+
+cleanup() {
+    print "Cleaning stuff up..."
+    helm uninstall kuberpult-local || print kuberpult was not installed
+    kind delete cluster || print kind cluster was not deleted
+}
+#trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
+trap cleanup INT TERM
+cleanup
+
+print 'creating kind cluster with a hostpath to share testdata...'
+kind create cluster --config=- <<EOF || print cluster exists
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraMounts:
+   - hostPath: $(pwd)/../../infrastructure/scripts/create-testdata/
+     containerPath: /create-testdata
+EOF
+
+export GIT_NAMESPACE=git
+
+LOCAL_EXECUTION=${LOCAL_EXECUTION:-false}
+print "LOCAL: $LOCAL_EXECUTION"
+
+
+print 'ensuring that the helm chart is build...'
+make all
+
+print installing ssh...
+./setup-cluster-ssh.sh
+
+print installing kuberpult...
+
+export IMAGE_REGISTRY=europe-west3-docker.pkg.dev/fdc-public-docker-registry/kuberpult
+
+if "$LOCAL_EXECUTION"
+then
+  print 'building cd service...'
+  WITH_DOCKER=true make -C ../../services/cd-service/ docker
+
+  print 'building frontend service...'
+  make -C ../../services/frontend-service/ docker
+else
+  print 'not building cd or frontend service...'
+fi
+
+cd_imagename=$(make --no-print-directory -C ../../services/cd-service/ image-name)
```

Comment:
Is that actually the correct version? We have this elaborate mechanism for calculating the IMAGE_TAG_CD and  IMAGE_TAG_FRONTEND but that doesn't seem to be used here at all.

Reply:
done